### ISSUE TITLE
Programmatically retrieve app service runtimes

### ISSUE THREAD

**Issue by EricJizbaMSFT** [Microsoft/vscode-azureappservice#724]
We keep having to manually update the list of runtimes used when creating a web app (See https://github.com/Microsoft/vscode-azureappservice/issues/723 as an example)

I would like us to re-investigate getting this list programmatically and file an issue on the app service team if it's not possible.

**Comment by EricJizbaMSFT** [comment]
Noticed that we list `10.10` as LTS, but I see this in the logs:
```
9:54:21 AM emjweb1: Detecting node version spec...
9:54:21 AM emjweb1: Using package.json engines.node value: >=6.9.1
9:54:21 AM emjweb1: Node.js versions available on the platform are: 4.4.7, 4.5.0, 6.2.2, 6.6.0, 6.9.3, 6.10.3, 6.11.0, 8.0.0, 8.1.0, 8.2.1, 8.8.1, 8.9.4, 8.11.2, 9.4.0, 10.1.0.
9:54:21 AM emjweb1: Resolved to version 10.1.0
```

Removing this from "engineering" milestone for re-triage as I think this is how we should fix issues like this.

**Comment by nturinski** [comment]
Let's 🐛  somebody to make this happen!!!! 🐞
